pkg/instancetype/infer/handler.go

```diff
@@ -68,21 +68,22 @@ func (h *handler) Instancetype(vm *virtv1.VirtualMachine) error {
 	if err != nil {
 		var ignoreableInferenceErr *IgnoreableInferenceError
 		if errors.As(err, &ignoreableInferenceErr) && ignoreFailure {
-			log.Log.Object(vm).V(logVerbosityLevel).Info("Ignored error during inference of instancetype, clearing matcher.")
-			vm.Spec.Instancetype = nil
+			log.Log.Object(vm).V(logVerbosityLevel).Info("Ignored error during inference of instancetype")
 			return nil
 		}
 		return err
 	}
 
+	// FIXME(lyarwood): @0xFelix I forget the context for this?
```

Comment:
@0xFelix something to discuss tomorrow, I can't recall why we do this but now we aren't modifying the spec is it going to cause issues?